CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 ## Changelog
 
 ### 2.3.2 (Next)
+* [#45](https://github.com/alexa-js/alexa-app-server/pull/45): Added support for ca chain certificates (#17) - [@tejashah88](https://github.com/tejashah88).
```

Comment:
Cosmetic: I usually write these changes like `[#45](...), [#17](...): ...`, just easier to grok.